trinity/plugins/registry.py

```diff
@@ -35,10 +35,14 @@ def is_ipython_available() -> bool:
         return True
 
 
-BUILTIN_PLUGINS = (
+COMMON_BUILTIN_PLUGINS: Tuple[BasePlugin, ...] = (
```

Comment:
Alternate name ideas:

- `CORE_PLUGINS`
- `BASE_PLUGINS`
- `SYSTEM_PLUGINS`

Reply:
+1 to core or base

Reply:
`BASE_PLUGINS` has my vote.